Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -905,6 +915,17 @@ public TValue SelectedValue
         /// </summary>
         [Parameter] public bool CloseOnSelection { get; set; } = true;
 
+        /// <summary>
+        /// Shows selected option(s) when presenting the searched values.
+        /// </summary>
+        [Parameter] public bool ShowSelected { get; set; }
```

Comment:
What exactly ShowSelected is doing. It's hard to tell from the naming.

Reply:
So, if you select a value autocomplete will no longer show it on the drop down options. This will still show it on the list....

Reply:
Ah, that makes sense. Yeah, it is a good feature.

Reply:
Right, the feature is basically what you're seeing on the print I posted on the first comment.
Just that is conflicts with the arrow/up selection, as the arrow/up selection has the same look.

Reply:
Maybe `SuggestSelected` I still think that's pretty similar to `Show` prefix eitherway. haha